bcda/bcdacli/cli_test.go

```diff
@@ -836,3 +834,110 @@ func getRandomPort(t *testing.T) int {
 
 	return listener.Addr().(*net.TCPAddr).Port
 }
+
+func CreateTestZipFile(file string) error {
+	zf, err := os.Create(file)
+	if err != nil {
+		return err
+	}
+	defer zf.Close()
+
+	w := zip.NewWriter(zf)
+	var files = []string{
```

Comment:
To keep isolation, should the caller of this function supply the files to create? So we change the `createTestZipFile` to `createTestZipFile(parent string, children ...string)` Then the caller of the function has an expectation on what file should exist in the zip.